javalanglib/src/main/scala/java/lang/Integer.scala

```diff
@@ -103,6 +103,21 @@ object Integer {
     ((t2 + (t2 >> 4) & 0xF0F0F0F) * 0x1010101) >> 24
   }
 
+  def highestOneBit(i: Int): Int = {
+    // ported from https://github.com/gwtproject/gwt/blob/master/user/super/com/google/gwt/emul/java/lang/Integer.java
+    if (i < 0) {
+      Integer.MIN_VALUE
+    } else if (i == 0) {
+      0
+    } else {
+      var rtn: Int = 1 << 30
+      while ((rtn & i) == 0) {
+        rtn >>= 1
+      }
+      rtn
+    }
```

Comment:
Oops, sorry, `i & -i` gives the `lowestOneBit`.

For `highestOneBit` the following should work, I believe:
```scala
if (i == 0) 0
else (1 << 31) >>> Integer.numberOfLeadingZeros(i)
```
Since `numberOfLeadingZeros` [will eventually be optimized as a built-in](https://github.com/scala-js/scala-js/issues/1275), this will be faster.